recipes/libass/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "libass" %}
+{% set version = "0.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libass/libass/releases/download/{{ version }}/libass-{{ version }}.tar.xz
+  sha256: 881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - autoconf
+    - automake
+    - pkg-config
+    - libtool
+  host:
+    - nasm
```

Comment:
Should this be in `build`?

Reply:
I think the package might use nasm to generate assembly for each platform separately.

Reply:
ie it doesn't cross-compile